File: DevSupportAndroid/src/main/java/com/sup/dev/android/views/views/ViewSpace.java
package com.sup.dev.android.views.views;

import android.content.Context;
import android.view.View;
import android.view.ViewGroup;

import com.sup.dev.android.tools.ToolsView;

public class ViewSpace extends View{

    private int w;
    private int h;

    public ViewSpace(Context context, int wDP, int hDP) {
        super(context);
        w = ToolsView.dpToPx(wDP);
        h = ToolsView.dpToPx(hDP);
    }

    @Override
    public void setLayoutParams(ViewGroup.LayoutParams params) {
        params.width = w;
        params.height = h;
        super.setLayoutParams(params);
    }
}

File: DevSupportAndroid/src/main/java/com/sup/dev/android/views/views/ViewSpace.kt
package com.sup.dev.android.views.views

import android.content.Context
import android.view.View
import android.view.ViewGroup
import com.sup.dev.android.tools.ToolsView


class ViewSpace(context: Context, wDP: Int, hDP: Int) : View(context) {

    private val w: Int = ToolsView.dpToPx(wDP.toFloat())
    private val h: Int = ToolsView.dpToPx(hDP.toFloat())

    override fun setLayoutParams(params: ViewGroup.LayoutParams) {
        params.width = w
        params.height = h
        super.setLayoutParams(params)
    }
}
